<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenCapturer</class>
 <widget class="QMainWindow" name="ScreenCapturer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>屏幕截图工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="captureTab">
       <attribute name="title">
        <string>截图设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="basicGroup">
          <property name="title">
           <string>基本设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="intervalLabel">
             <property name="text">
              <string>截图间隔：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="intervalSpin"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="minuteLabel">
             <property name="text">
              <string>分钟</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="monitorLabel">
             <property name="text">
              <string>显示器：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QComboBox" name="monitorCombo"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="directoryLabel">
             <property name="text">
              <string>保存目录：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="directoryEdit"/>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="browseButton">
             <property name="text">
              <string>浏览...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="formatLabel">
             <property name="text">
              <string>图片格式：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="formatCombo"/>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="qualityLabel">
             <property name="text">
              <string>质量：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QSpinBox" name="qualitySpin"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="optionsGroup">
          <property name="title">
           <string>选项</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="clipboardCheck">
             <property name="text">
              <string>截图时复制到剪贴板</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="showBordersCheck">
             <property name="text">
              <string>显示显示器边框</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="immediateCheck">
             <property name="text">
              <string>启动时立即截图</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="openFolderCheck">
             <property name="text">
              <string>截图后打开文件夹</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="notificationsCheck">
             <property name="text">
              <string>显示系统通知</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="minimizeToTrayCheck">
             <property name="text">
              <string>最小化到系统托盘</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QCheckBox" name="enablePreviewCheck">
             <property name="text">
              <string>启用截图预览</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>开始定时截图</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string>停止截图</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="captureNowButton">
            <property name="text">
             <string>立即截图</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="aboutButton">
            <property name="text">
             <string>关于</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statusTab">
       <attribute name="title">
        <string>状态信息</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="statusGroup">
          <property name="title">
           <string>运行状态</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="statusTitleLabel">
             <property name="text">
              <string>当前状态：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="statusLabel">
             <property name="text">
              <string>就绪</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="counterTitleLabel">
             <property name="text">
              <string>截图计数：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="counterLabel">
             <property name="text">
              <string>已截图: 0 张</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="nextCaptureTitleLabel">
             <property name="text">
              <string>下次截图：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="nextCaptureLabel">
             <property name="text">
              <string>下次截图: --</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="timeTitleLabel">
             <property name="text">
              <string>当前时间：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="timeLabel">
             <property name="text">
              <string>--</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="previewGroup">
          <property name="title">
           <string>截图预览</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="previewLabel">
             <property name="text">
              <string>预览区域</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>